rotkehlchen/tests/api/test_balances.py

```diff
@@ -1287,3 +1287,44 @@ def test_solana_balances_multiple_accounts(
     assert result['assets']['solana/token:EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v']['amount'] == '160556.002808'  # USDC  # noqa: E501
     assert result['assets']['solana/token:2zMMhcVQEXDtdE6vsFS7S7D5oUodfJHE8vd1gnBouauv']['amount'] == '370905.247021'  # PENGU  # noqa: E501
     assert result['assets']['solana/token:WLFinEv6ypjkczcS83FZqFpgFZYwQXutRbxGe7oC16g']['amount'] == '100122.11'  # WLFI  # noqa: E501
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12', '0x9531C059098e3d194fF87FebB587aB07B30B1306']])  # noqa: E501
+def test_blockchain_balances_specific_addresses(
+        rotkehlchen_api_server: 'APIServer',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    """Test that querying blockchain balances for specific addresses works correctly"""
+
+    # Query only the first address
+    response = requests.get(
```

Comment:
```suggestion
    partial_result = assert_proper_sync_response_with_result(requests.get(
```
same suggestion for the other request below.